demo/js/demo.js

```diff
@@ -718,9 +720,7 @@ window.onload = function() {
     });
 
     grid.addProperties({
-        readOnly: false,
-        noDataMessage: 'No Data to Display',
-        showFilterRow: true
+        readOnly: false
```

Comment:
It seems to be an attempt to bail from event generation, making the grid "read-only" by ignoring all mouse and keyboard events. I'm not sure it is everywhere it needs to be. Since you're into events this week, could you make sure it is fully implemented?